docs/user-guide.md

```diff
@@ -785,6 +787,11 @@ There are some caveats:
 * When used in the main thread, the `display` function automatically uses
   the current `<script>` tag as the `target` into which the content will
   be displayed.
+
+    - If the `<script>` tag has the `target` attribute, the element on the page
+    with that ID (or which matches that selector) will be used to display
+    the content instead.
+
 * When used in a worker, the `display` function needs an explicit
```

Comment:
this is actually inaccurate ... the `target` is retrieved and created automatically when `current_target()` is used in Python, returning the right ID ... this was mistakenly left there but nothing needs a target *unless* it's a different element (or companion element).

Reply:
I'm not sure I quite follow, sorry... this change in the docs is trying to say:

The following tag puts _"Hello!"_ in the Element with id "bar":
```html
<script type="py" target="bar">
    display("Hello!")
</script>
```
Is that an accurate description of things? Perhaps I've misunderstood... or maybe I just worded this PR poorly.

Reply:
the line I've commented is out of this PR ... it's exactly this paragraph:

>  When used in a worker, the `display` function needs an explicit

this is not true anymore as `display` works without any target out of the box in workers too.

Since you've been so kind to help improving docs, I've pointed out there's one part of the docs that is currently misleading.

Reply:
Ah, I'm so sorry, got confused by GitHub's UI. Addressed in [7493034](https://github.com/pyscript/docs/pull/17/commits/74930343330adac069469669520bac40b27f9bf0).